contrib/google.golang.org/grpc/option.go

```diff
@@ -58,11 +58,9 @@ func defaults(cfg *config) {
 	cfg.traceStreamCalls = true
 	cfg.traceStreamMessages = true
 	cfg.nonErrorCodes = map[codes.Code]bool{codes.Canceled: true}
-	// cfg.analyticsRate = globalconfig.AnalyticsRate()
+	// cfg.spanOpts = append(cfg.spanOpts, tracer.AnalyticsRate(globalconfig.AnalyticsRate()))
```

Comment:
Does this comment need to stay around? Can it just be deleted?

Reply:
This comment exists for all other integrations in the contrib package which don't inherit the analytics rate from the global analytics rate (e.g. [gorm](https://github.com/DataDog/dd-trace-go/blob/4da037f6a13f8886a843cf63514b28d296217718/contrib/gopkg.in/jinzhu/gorm.v1/option.go#L29)). I'd like to keep it just to stay consistent with the other contrib packages (although perhaps we could remove the comments from all contrib packages in a separate PR).

What do you think ? 🙂

Reply:
When I implemented this feature, I left this here as a note that this integration does not have span analytics _on purpose_ and that it may be enabled later on. If you think it's better to remove them, feel free to do so but please remove them everywhere.

Reply:
Oh in that case that's fine no need to remove it. I didn't realize it was across a number of integrations